source/locales/zh-CN.json

```diff
@@ -108,9 +108,9 @@
   "Notes": "注意",
   "Operating System": "操作系统",
   "trisquel": "自由软件基金会（FSF）认可的，对用户友好的 GNU/Linux 发行版",
-  "operating-systems-notes": "<p>Apple, Google, and Microsoft are allegedly a part of PRISM. Their proprietary operating systems cannot be trusted to safeguard your personal information from the NSA. We have two free alternatives: <strong><a href='https://www.gnu.org/distros/free-distros.html'>GNU/Linux</a></strong> and <strong><a href='https://en.wikipedia.org/wiki/Comparison_of_BSD_operating_systems'>BSD</a></strong>.</p><p class='hideable'>GNU/Linux has a much larger community to help you with the transition. It’s recommended that you begin your explorations by looking for a <a href='http://distrowatch.com/'>GNU/Linux distribution</a> that suits your needs. Additionally the <a href='https://www.gnu.org/distros/free-distros.html'>Free Software Foundation hosts a list of completely Free distributions</a>.</p><p><strong>Debian</strong> has a long tradition of software freedom. Contributors have to sign a social contract and adhere to the ethical manifesto. Strict inclusion guidelines make sure that only certified open source software gets packaged in the main repositories. <p><strong>Fedora</strong> is a community edition that serve as the stable basis for enterprise ready GNU/Linux distributions with commercial support. Companies all over the world trust Red Hat Inc. because of their transparency throughout the whole development process.</p><p>Canonical’s Ubuntu is not recommended by PRISM Break because it contains <a href='https://www.eff.org/deeplinks/2012/10/privacy-ubuntu-1210-amazon-ads-and-data-leaks'>Amazon ads and data leaks</a> by default. GNU/Linux distributions based on Ubuntu are also currently not recommended due to <a href='https://github.com/nylira/prism-break/issues/334'>several other reasons.</a></p>",
+  "operating-systems-notes": "<p>Apple、Google 和 Microsoft 是 PRISM 项目的一部分。他们专有的操作系统不能被信任来保护你的个人信息不被泄露给美国国家安全局 NSA。</p><p>这留给我们两个免费替代品： <strong><a href='https://www.gnu.org/distros/free-distros.html'>GNU/Linux</a></strong> 和 <strong><a href='https://en.wikipedia.org/wiki/Comparison_of_BSD_operating_systems'>BSD</a></strong> .</p> <p>GNU/Linux 有一个更大的社区来帮助你的转换过程。我们建议新手先从适合你的 <a href='http://distrowatch.com/'>GNU/Linux 发行版</a>开始。</p><p>Canonical&rsquo;s Ubuntu 并不推荐，因为其默认配置存在<a href='https://www.eff.org/deeplinks/2012/10/privacy-ubuntu-1210-amazon-ads-and-data-leaks'>亚马逊广告和数据泄漏漏洞</a>如果你已经用上了 Ubuntu，你可以从 Fix Ubuntu 网站获得亡羊补牢的建议措施。</p>",
```

Comment:
This is the only place that looks suspicious, in that it was much larger previously. I don't know Chinese and can't merge properly so I'm going to revert this.

Reply:
Managed to merge this too.